app/controllers/manage_cmr/groups_controller.rb

```diff
@@ -192,6 +212,14 @@ def group_params
     params.require(:group).permit(:name, :description, :provider_id, members: [])
   end
 
+  # def groups_index_params
```

Comment:
Might be worth leaving a note about what did/didn't work here for when the ticket(s) to implement these do get worked.

Reply:
added in index action